Task_2_2

oleksiiKhlystun opened the pull request:
Task_2_2

oleksiiKhlystun committed:
Task_2_2

Ruslan4 commented on Task_2_2a_Cons_Dr_Methods/Box.cs:
```diff
@@ -0,0 +1,63 @@
+        //and auto-implement properties for a symbol of a given set of valid characters (*, + ,.) to be used for the border 
+        //and a message inside the box
+
+        public int Width { get; set; } = Convert.ToInt32(Console.ReadLine());
```
it would be better to transfer keyboard input logic (Convert.ToInt32(Console.ReadLine());) to  static void Main(string[] args)

oleksiiKhlystun replied:
ok

Ruslan4 commented on Task_2_2a_Cons_Dr_Methods/Box.cs:
```diff
@@ -0,0 +1,63 @@
+        //Change the message in the method to return the Box square
+        //Use Console.SetCursorPosition() method
+        //Trim the message if necessary
+        private int draw(int Height, int Width, string Simbol, int Left, int Top)
```
`private int draw(int height, int width, string simbol, int left, int top)`

Ruslan4 commented on Task_2_2a_Cons_Dr_Methods/Box.cs:
```diff
@@ -0,0 +1,63 @@
+        //Change the message in the method to return the Box square
+        //Use Console.SetCursorPosition() method
+        //Trim the message if necessary
+        private int draw(int Height, int Width, string Simbol, int Left, int Top)
```
```suggestion
        private int draw(int height, int width, string simbol, int left, int top)
```

oleksiiKhlystun replied:
ok

oleksiiKhlystun committed:
chanches about uppercase and lowercase properties

oleksiiKhlystun committed:
changes about ReadLine to Main

changes about Convert.ToInt32(Console.ReadLine()); to static void Main(string[] args)

oleksiiKhlystun committed:
f

oleksiiKhlystun commented:
ok
